block/base/options.go

```diff
@@ -0,0 +1,80 @@
+package base
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/skip-mev/block-sdk/block"
+)
+
+// LaneOption defines a function that can be used to set options on a lane.
+type LaneOption func(*BaseLane)
+
+// SetAnteHandler sets the ante handler for the lane.
+func SetAnteHandler(anteHandler sdk.AnteHandler) LaneOption {
+	return func(l *BaseLane) { l.cfg.AnteHandler = anteHandler }
+}
+
+// SetPrepareLaneHandler sets the prepare lane handler for the lane. This handler
+// is called when a new proposal is being requested and the lane needs to submit
+// transactions it wants included in the block.
+func SetPrepareLaneHandler(prepareLaneHandler PrepareLaneHandler) LaneOption {
```

Comment:
I like naming options like this: "WithPrepareLaneHandler" instead of using "Set".

I think it makes it a bit cleaner to reason about what the options are doing.  Maybe just pref tho

Reply:
I was just looking at the way base app lets u set options, but i agree, will change that